src/Common/tests/System/Net/Capability.Sockets.cs

```diff
@@ -47,5 +47,10 @@ public static bool IPv4Support()
         {
             return Socket.OSSupportsIPv4;
         }
+
+        public static bool UnixDomainSocketsSupport()
+        {
+            return !RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
```

Comment:
You cannot access internal methods from either apps or functional tests. Why would you add it as an internal method if it can't be used in either apps or tests?